app/js/controllers/dev.js

```diff
@@ -0,0 +1,21 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import CameraLatestImageView from 'js/views/dev/camera-latest-image';
+
+export default class DevController extends Controller {
```

Comment:
Didn't want to pollute existing views/controllers with such temporal stuff and think that we may need more such views in the future, so here is the idea with Dev controller that can serve various dev/demo related views.

How does it look to you?